### DISCUSSION TITLE
Discord.ts

### DISCUSSION THREAD

**Discussion by cristinnel** [jef/streetmerchant#1141] [Help]
Unable to remove
 Hey guys,ill really apreciate when you have time and want to help,i manage to work with the notifications on discord server(only me on it :D just for the sound ) im running the run test,its show in discord and desktop notification,the problem is on discord it is like this: ill atach a picture ,any ideea what should i delete in the discord.ts file,so it dosent show anymore test:brand,test:series,test:model and the url with the example ? Thank you very much! Stay safe.
![Discord_1lwPo1PBmR](https://user-images.githubusercontent.com/75442432/101088602-3be23d00-35b4-11eb-8c06-db29f01d72e7.png)

**Comment by MIKEBURNA** [comment]
What? You run the test? this is the test notification. fix your search parameters to get real offers and it will show up correctly.

**Comment by cristinnel** [comment]
> What? You run the test? this is the test notification. fix your search parameters to get real offers and it will show up correctly.

oh yea thats the problem, i dont know how :D thanks anyway 👍
